ui/src/App.js

```diff
@@ -102,7 +94,6 @@ class App extends Component {
           <FieldSearch
```

Comment:
FieldSearch component should be renamed to Search (FieldSearch.js -> Search.js).
fields prop -> searchResults.
handleFieldSearchChange -> handleSearch

Can be in same PR. :)